e2etest/bgp_test.go

```diff
@@ -82,11 +82,11 @@ var _ = ginkgo.Describe("BGP", func() {
 		{
 			name: frrEBGP,
 			nc: frrconfig.NeighborConfig{
-				ASN:      EBGPAsn,
+				ASN:      IBGPAsn,
```

Comment:
Can you add a message to the commit explaining why we are removing the view support and what are the tradeoffs here?